infrasim/cloud_img.py

```diff
@@ -0,0 +1,88 @@
+'''
+*********************************************************
+Copyright @ 2015 EMC Corporation All Rights Reserved
+*********************************************************
+'''
+
+import os
+import string
+import random
+from infrasim import run_command
+
+
+# def load_backing_file we can add load backing file from cloud image offical website later
+
+def gen_qemuimg(boot_img):
+    status, output = run_command("qemu-img create -f qcow2 -o backing_file=/home/infrasim/jenkins/data/"
+                                 "ubuntu-16.04-server-cloudimg-amd64-120G.org.bak cloudimgs/{}".format(boot_img))
+    return str(os.getcwd() + "/cloudimgs/") + boot_img
```

Comment:
force to use current folder is not a good choice. it's better to let user to specify it.

Reply:
It's a tmp folder, user should not care the path.